desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -333,10 +333,10 @@ void DownloadFromDCWidget::on_downloadCancelRetryButton_clicked()
 	QString product(ui.product->currentText());
 	//
 	// We shouldn't do this for memory dumps.
-	if ((product == "OSTC 3" || product == "OSTC 3+" ||
-			product == "OSTC cR" || product == "OSTC Sport" ||
-			product == "OSTC 4") && ! data->saveDump())
-	ostcFirmwareCheck = new OstcFirmwareCheck(product);
+	if ((product == "OSTC 3" || product == "OSTC 3+" || product == "OSTC cR" || product == "OSTC Sport" ||
+		product == "OSTC 4") && ! data->saveDump()) {
+			ostcFirmwareCheck = new OstcFirmwareCheck(product);
+	}
```

Comment:
should be in the lines of:

```
if ((product == "OSTC 3" || product == "OSTC 3+" || product == "OSTC cR" ||
     product == "OSTC Sport" || product == "OSTC 4") && !data->saveDump()) {
	ostcFirmwareCheck = new OstcFirmwareCheck(product);
}
```
notice how `product` on the second line is indented with spaces, while the `ostcFirmwareCheck` is indented with a tab.
also fixes extra space between `!` and `data->saveDump()`.